src/client/rspamc.cxx

```diff
@@ -915,26 +977,36 @@ rspamc_metric_output(FILE *out, const ucl_object_t *obj)
 					colorized_action = fmt::format(fmt::emphasis::bold, ucl_object_tostring(elt));
 					break;
 				}
-				fmt::print(out, "Action: {}\n", colorized_action);
+				fmt::print(out, fmt::runtime(humanreport ? ",Action={}" : "Action: {}\n"), colorized_action);
 			}
 
-			fmt::print(out, "Spam: {}\n", emphasis_argument(act.value() < METRIC_ACTION_GREYLIST ?
-											  "true" : "false"));
+			fmt::print(out, fmt::runtime(humanreport ? ",Spam={}" : "Spam: {}\n"),
```

Comment:
I'd suggest to convert `fmt::runtime` to a simple `if...else` to keep fmt constexpr providing better compile time checks.